obip-0008.md

```diff
@@ -0,0 +1,146 @@
+<pre>
+OBIP: 8
+Title: 3rd Party Moderator Verification
+Author: Tyler Smith <tyler@ob1.io>
+Discussions-To: Github Issues
+Status: Draft
+Type: Standards Track
+Created: 02/28/2018
+Copyright: MIT
+</pre>
+
+## Abstract
+A specification for a system allowing 3rd parties to provide a list of moderators that it has verified in some way, with the nature and extent of the verifications being defined by the 3rd party itself.
+
+## Motivation
+Allow buyers on OpenBazaar to have more information about the identity and reputation of moderators for given listings. If a 3rd party can provide accurate verifications of the moderator's identity, knowledge, fairness, and/or reliability then users can make safer decisions.
+
+## Verified Moderators Endpoint
+
+To provide verified moderators a provider must have a public endpoint returning data with the following schema:
```

Comment:
I think it would be worth pointing out that none of the data is optional. There must be at least one type. The moderators array can be empty, but must be returned even if it's empty.